mods/blocks/book.lua

```diff
@@ -4,36 +4,54 @@ local esc = minetest.formspec_escape
 local formspec_size = "size[8,8]"
 
 local function formspec_core(tab)
-	if tab == nil then tab = 1 else tab = tostring(tab) end
-	return "tabheader[0,0;book_header;" ..
-		esc(S("Write")) .. "," ..
-		esc(S("Read")) .. ";" ..
-		tab  ..  ";false;false]"
+	if tab == nil then
+		tab = 1
+	else
+		tab = tostring(tab)
+	end
+	return "tabheader[0,0;book_header;" .. esc(S("Write")) .. "," .. esc(S("Read")) .. ";" .. tab .. ";false;false]"
 end
 
 local function formspec_write(title, text)
-	return "field[0.5,1;7.5,0;title;" .. esc(S("Title:")) .. ";" ..
-			esc(title) .. "]" ..
-		"textarea[0.5,1.5;7.5,7;text;" .. esc(S("Contents:")) .. ";" ..
-			esc(text) .. "]" ..
-		"button_exit[2.5,7.5;3,1;save;" .. esc(S("Save")) .. "]"
+	return "field[0.5,1;7.5,0;title;"
+		.. esc(S("Title:"))
+		.. ";"
+		.. esc(title)
+		.. "]"
+		.. "textarea[0.5,1.5;7.5,7;text;"
+		.. esc(S("Contents:"))
+		.. ";"
+		.. esc(text)
+		.. "]"
+		.. "button_exit[2.5,7.5;3,1;save;"
+		.. esc(S("Save"))
+		.. "]"
```

Comment:
this is entirely wonky

Reply:
yeah, can't say i'm a fan of dots in front - i think this can be changed though

Reply:
it can't be changed, at least not with this formatter - https://github.com/JohnnyMorganz/StyLua